code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -1083,8 +1083,18 @@
 		target_data["ref"] = stretcher_ref
 
 		var/mob/living/carbon/human/occupant = stretcher.buckled_mob
+		var/obj/structure/closet/bodybag/cryobag = stretcher.buckled_bodybag
+		if(!occupant && cryobag)
+			for(var/atom/movable/AM in cryobag)
+				if(ishuman(AM))
+					occupant = AM
+					break
```

Comment:
```suggestion
			for(var/mob/living/carbon/human in cryobag)
				occupant = AM
				break
```
there's no point doing two typechecks here

alternatively, this cna be expressed as
```dm
occupant = locate() in cryobag
```